### PR TITLE
fix/npm license

### PR DESCRIPTION
**Pull request by baywet**
SPDX was wrong, see the id here https://spdx.org/licenses/Apache-2.0.html
This is in turns causing clearlydefined to fail, which in turns triggers warnings in our internal pipelines. https://clearlydefined.io/definitions/npm/npmjs/@std-uritemplate/std-uritemplate/1.0.5

### CODE REVIEW

**Review by andreaTP — APPROVED** [review]
LGTM, thanks!

**Comment by andreaTP** [comment]
Do you need a release @baywet ?
Or are you going to do it yourself?

**Comment by baywet** [comment]
Thanks @andreaTP !
I was about to ask actually. I'm happy to do it myself, I just don't remember how this repo is setup.
Do I only need a tag? Or is it more complex than that?

**Comment by baywet** [comment]
I don't mind running it. The only things that's not clear from reading the script and associated doc is whether I need to include the v prefix or not?
Once you give me the answer I'll probably stand another docs PR, so when I forget in the future and need to come back to it, I don't ask the same question again :)

**Comment by andreaTP** [comment]
no `v` prefix. we should write it down ...

**Comment by andreaTP** [comment]
The Go release automatically adds the `v` prefixed tag:
https://github.com/std-uritemplate/std-uritemplate/blob/42ff6d3a328821d11490c0daa49cc897497035a7/.github/workflows/release.yml#L100

**Comment by baywet** [comment]
Thank you for the additional information.
#235

**Comment by baywet** [comment]
just to provide feedback on the script itself: I'm on windows here, and wsl is not setup with commit/tag signature properly. So what I ended up doing is run the commands in the script manually.
Also the difference is that -S implies -s on windows, which I don't think it does on linux (-S being sign the commit, -s being add the signed by foo in the commit header, GitHub needing both as far as I understand to add the checkmark)

Long story short:
- I don't think we need a windows equivalent script
- We should probably add the -S -s arguments for the tag/commits commands in case the person running that is not globally configured. But if we do, if they don't have a signing agent/key configured, they'll get stuck. So I'm not sure what you want to do here?
